### ISSUE TITLE
'godot.builtins.Vector2' object has no attribute 'direction_to'

### ISSUE THREAD

**Issue by keenfoong** [touilleMan/godot-python#236]
Hi, running into an issue where I cant run a base function in Vector2

```
self.velocity = self.global_position.direction_to(self.get_global_mouse_position())
AttributeError: 'godot.builtins.Vector2' object has no attribute 'direction_to'
```

Here is the doc:
[https://docs.godotengine.org/en/stable/classes/class_vector2.html?highlight=move_toward#class-vector2-method-direction-to](https://docs.godotengine.org/en/stable/classes/class_vector2.html?highlight=move_toward#class-vector2-method-direction-to)

Running in GDscript works fine. From what it looks like, godot python isn't quite there yet?

**Comment by touilleMan** [comment]
Thanks for pointing this out !
`Vector2/3.direction_to` has been added in GDNative 1.2 and forgot to add them

**Comment by touilleMan** [comment]
FYI I'm currently rewriting the builtins generation code (see #242) to fix the issue.
This PR is more complicated than anticipated given `pythonscript\godot\_hazmat\gdnative_api_struct.pxd` must be re-generated every time `gdnative_api.json` changes (it used to be semi-automatically generated with a hackily patched autopxd2, so this had to change to something cleaner...)

**Comment by gustavi** [comment]
This issue is fixed in version `0.50.0`.

You can try the following code, it works fine for me : 

```python
from godot import Node2D, Vector2, exposed


@exposed
class Issue236(Node2D):

    def _ready(self):
        """
        Called every time the node is added to the scene.
        Initialization here.
        """
        x = Vector2(1, 0)
        y = Vector2(6, -4)

        print(x.direction_to(y))

```

Output : 

```
Godot Engine v3.2.3.stable.official - https://godotengine.org
[...]
Pythonscript 0.50.0 (CPython 3.8.5.final.0)
[...]
Loading resource: res://Issue236.tscn
Loading resource: res://Issue236.py
<Vector2(x=0.7808687686920166, y=-0.6246950626373291)>
```

**Comment by touilleMan** [comment]
good ;-)
